src/main/java/uk/gov/justice/digital/delius/data/api/Disability.java

```diff
@@ -17,4 +18,5 @@
     private LocalDate startDate;
     private LocalDate endDate;
     private String notes;
+    private List<Provision> provisions;
```

Comment:
can you think of a better name for this? I can guess this means provisions to mitigate the disability but not 100% sure. Don't worry if you can't think of a better name.

As a matter of interest do you have an example for a provision?

Reply:
On the NDelius frontend, these are called Adjustments rather than provisions - I've gone with naming the same as the database table, as I think there is precedent for that (e.g. Probation Area is Provider on the front end, but we still call it Probation Area here)

Examples could be something physical like a wheelchair ramp or stair lift, or additional help/ assistance.